scripts/utils/getAliases.ts

```diff
@@ -30,7 +30,10 @@ export default function getAliases(cwd = process.cwd(), format: "webpack" | "jes
                 aliases[name] = url;
             } else {
                 const name = n.replace("/*", "/(.*)$");
-                const url = p.replace("./", "<rootDir>/").replace("/*", "/$1");
+                const url = (p.startsWith("./")
+                    ? p.replace("./", "<rootDir>/")
+                    : "<rootDir>/".concat(p)
+                ).replace("/*", "/$1");
```

Comment:
Maybe put a comment here? I forget why this was even necessary

Reply:
Tried undoing it and I guess it wasn't even necessary